bin/GameIndex.dbf

```diff
@@ -36270,7 +36264,7 @@ Name   = Kessen 2
 Region = NTSC-U
 Compat = 5
 [patches = 53A803AF]
-	comment=COP2 flag instance patch by refraction
+	comment=COP2 flag instance patch by Refraction
```

Comment:
Isn't it "refraction"?

Reply:
It is "refraction" I don't use a capital R